src/shared/services/auth/services/access-token-parser/access-token-parser.js

```diff
@@ -13,7 +13,7 @@ function decodeToken(token) {
   }
 }
 
-export default function parseAccessToken(token) {
+export default function accessTokenParser(token) {
```

Comment:
:+1:

Reply:
I don't agree here.
Function names should indicate an action, not a 'thing.'
`parseAccessToken` conveys an action, 'accessTokenParser` conveys a 'thing'.
Using the same name for file/folder is a good thing, though 👍